www/include/reporting/dashboard/csvExport/csv_ServiceLogs.php

```diff
@@ -46,17 +46,17 @@
 session_start();
 session_write_close();
 
-/*
- * DB connexion
- */
+// DB connexion
 $pearDB = new CentreonDB();
 $pearDBO = new CentreonDB("centstorage");
 
 $sid = session_id();
 if (!empty($sid) && isset($_SESSION['centreon'])) {
     $oreon = $_SESSION['centreon'];
-    $query = "SELECT user_id FROM session WHERE user_id = '".$pearDB->escape($oreon->user->user_id)."'";
-    $res = $pearDB->query($query);
+    $res = $pearDB->query(
+        "SELECT user_id FROM session " .
+        "WHERE user_id = '" . $pearDB->escape($oreon->user->user_id) . "'"
```

Comment:
In this case, using a prepare statement is hightly recommended

Reply:
Refacto the whole code is out of my scope...  ;)